controllers/managedmcg_controller.go

```diff
@@ -156,32 +160,23 @@ func (r *ManagedMCGReconciler) Reconcile(_ context.Context, req ctrl.Request) (c
 
 func (r *ManagedMCGReconciler) reconcilePhases() (reconcile.Result, error) {
 	r.Log.Info("reconciliation phases initiated")
-	foundAddonDeletionKey := r.verifyAddonDeletionKey() // TODO
-	r.updateNoobaaComponentStatus()
+	foundAddonDeletionKey := r.verifyAddonDeletionKey()
+	r.updateComponentStatus()
```

Comment:
I thought we agreed on the previous name when the refactor PR went in?

Reply:
need to update status for Prometheus and AlerManager along with Noobaa, because of that method name `updateNoobaaComponentStatus()` dosnt make sense.

Reply:
Okay.